packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -69,6 +73,9 @@ class AndroidCameraCameraX extends CameraPlatform {
   @visibleForTesting
   CameraInfo? cameraInfo;
 
+  /// The [CameraControl] instance that corresponds to the [camera] instance.
+  CameraControl? cameraControl;
+
```

Comment:
Decided to save this whenever the related `camera` is updated versus making async calls to retrieve it every time that it's needed.